utility/servicer/module_test.go

```diff
@@ -248,6 +264,7 @@ func testServicerConfig() configs.ServicerConfig {
 		Address:   testServicer1.Address,
 		Services: map[string]*configs.ServiceConfig{
```

Comment:
Thank you for taking the time to update the tests based on all my requests. Goes a long way!